templates/web/base/govuk/fields.html

```diff
@@ -268,7 +268,8 @@ <h2 class="govuk-fieldset__heading">
         <span class="govuk-visually-hidden">Error:</span> [% error %]
     </span>
   [% END %]
-  <select class="govuk-select" id="[% field.id %]" name="[% field.html_name %]">
+  <select class="govuk-select[% ' js-autocomplete__input--show-all-value' IF field.get_tag('autocomplete') %]" id="[% field.id %]" name="[% field.html_name %]">
+    <option value=""></option>
```

Comment:
Thanks @dracos that makes sense =) and the code on `FixMyStreet.js` look cleaner.
I just checked and is working for me. 

Because we are deploying it across all the WW cobrands should I deploy directly to `master` or should we show it to the client on `commercial-staging` first?

Reply:
I think we can just deploy it, it's an improvement and shouldn't have any bad effects